TACC/Core-Portal
# Task/FP-1321: Reindex job archive path.

## Overview: ##
This fixes indexing for file operations and jobs.

## Related Jira tickets: ##

* [FP-1321](https://jira.tacc.utexas.edu/browse/FP-1321)

## Summary of Changes: ##
The client.files.list() function seemed to be returning the raw string of a function call as the date for `lastModified` (e.g.  `'lastModified': datetime.datetime(2022, 5, 23, 16, 37, 49, tzinfo=tzoffset(None, -18000))` )
This was failing silently in our end and causing the file item to be not indexed (specifically `walk_levels()` used by `agave_indexer()` failed to return the correct file data). This function is used to index all of our file operations and jobs.
 
Once I casted the `offset` and `limit` to integers, it returned the proper `datetime` string.

This seems to be a bug on the `agavepy` side?

## Testing Steps: ##
1. Compress a file and try searching it from the My Data search for the target system.
2. Ensure that the file shows up in the search result.

## UI Photos:

## Notes: ##
